forward/forwarder.go

```diff
@@ -50,6 +50,7 @@ type Options struct {
 	RerouteRetries bool
 	RetrySchedule  []time.Duration
 	Timeout        time.Duration
+	Headers        []byte
```

Comment:
Sure. So TChannel itself only has a notion of "bag of bytes" for arg2. The encoding of headers from `map[string]string` to the `[]byte` is encoding dependent. The specific encoding details can be found at: